code/modules/jobs/job_types/_job.dm

```diff
@@ -174,6 +174,9 @@
 
 	var/pda_slot = SLOT_BELT
 
+	var/jumpsuit
```

Comment:
what's the rationale for this?

Uniform should have worked fine. Only usecase I can currently think of where this would be req'd would be if a job had a default value of a skirt and you had to "opt in" to choosing the jumpsuit instead, which will never happen in practice (tfw no maid cafe job)

Reply:
See post below

Reply:
I'm not sure what you mean by "uniform should have worked fine" either, considering uniform was never touched? Uniform is untouched by jumpsuit/jumpskirt if unset, so this doesn't break any other outfits, either.

Reply:
It should be uniform's default is the jumpsuit type BUT if you have the pref set to skirt and there's an available option for said skirt then uniform is set to skirt instead. 

Right now the grey jumpsuit is the uniform's default (you've removed the uniform var from subtypes so it inherits) then it runs through the check twice for both jumpsuits and skirts. It does not need to do that when you can just set the jumpsuit as `uniform` with the implication that if there's neither a skirt preference or a skirt option, just run normally with the defined jumpsuit.